packages/core/src/browser/browser-clipboard-service.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2019 TypeFox and others.
+ * Copyright (C) 2019 RedHat and others.
```

Comment:
@theia-ide/ip-help I think it is fine, right? I've never merged these files or we should always preserver copyright even while copying code from each other PRs?